templates/api/knowledge-panels/environment/manufacturing_place.tt.json

```diff
@@ -0,0 +1,39 @@
+{
+    "parent_panel_id": "root",
+    "type" : "score",
+    "level" :"info",
+    "topics": [
+        "environment"
+    ],
+    "evaluation": "neutral",
+    "expanded": "yes",
+    "title_element": {
+        "title": "[% lang('manufacturing_places_s') FILTER ucfirst %]",
+        [% IF panel.packager_code_data.cc == 'fr' %]
+            "subtitle": "[% panel.packager_code_data.commune %] - [% display_taxonomy_tag('countries', 'en:france') %]",        
+        [% ELSIF panel.packager_code_data.cc == 'ch' %]
+            "subtitle": "[% panel.packager_code_data.full_address %]",
+        [% ELSIF panel.packager_code_data.cc == 'es' %]
+            "subtitle": "[% panel.packager_code_data.provincia_localidad %]",
+        [% ELSIF panel.packager_code_data.cc == 'uk' %]
+            "subtitle": "[% panel.packager_code_data.district %]",             
+        [% END %]
```

Comment:
Wow that seems not very scalable ! Why do we have different fields name based on the country ?

We should at least hide it in a perl function…

Reply:
Well it's complicated: each country has a registry of packaging codes to companies, but each of them has completely different fields for them. So far we chose to include all the fields they have, but we have not attempted to map fields to common names.

Reply:
So here in fact I think it's more of a display choice: for each country, which field do we want to use as title, and which field do we want to use as subtitle. We can't always use the same field (e.g. the company name, or the city + the country) because those fields may not be available for that country.